spec/system/consumer/split_checkout_spec.rb

```diff
@@ -502,6 +502,42 @@
           end
         end
       end
+
+      describe "hidding a payment method with a default rule" do
+        let!(:tagged_customer) { create(:customer, user: user, enterprise: distributor) }
+        let!(:hidden_payment) {
```

Comment:
```suggestion
        let!(:hidden_method) {
```

Since we use the word *payment* for something else, this may be confusing.